src/tables/sandbox.js

```diff
@@ -1,111 +1,104 @@
-let dynamo = require('./dynamo')
-let promisify = require('./promisify-object')
-let parallel = require('run-parallel')
+const promisifyObject = require('./promisify-object')
+const parallel = require('run-parallel')
+
+const stagingTables = tbl => !tbl.includes('-production-')
```

Comment:
I think we want to keep the filter regarfing arc-sessions table in place in case old school arc users still opt-into using the dynamodb sessions table for session support.

Reply:
How it works in production:

- at deploy time, [Architect compiles an AWS SSM Parameter for each table that maps the user-provided table name to the logical ID generated by CloudFormation](https://github.com/architect/package/blob/master/src/visitors/globals/ssm.js#L6-L26)
- at runtime in prod (and also staging), [arc/functions looks up these parameters to do a kind of service discovery](https://github.com/architect/functions/blob/master/src/tables/index.js#L42-L49)
- [the service discovery results are then fed into a db 'factory' module](https://github.com/architect/functions/blob/master/src/tables/factory.js)

As for a pseudocode test, first context:

- When [sandbox starts up, it creates an `arc-sessions` table (just in case)](https://github.com/architect/sandbox/blob/master/src/tables/_init.js#L20-L28)
- At runtime, if you use the [`arc.http.session` helper functions](https://arc.codes/docs/en/reference/runtime/node#arc.http.session), these functions [check for a specific environment variable before determining which location to read from](https://github.com/architect/functions/blob/master/src/http/session/read.js) and [write to sessions](https://github.com/architect/functions/blob/master/src/http/session/write.js), which is (I believe) either JSON web token or using DynamoDB to track sessions.

... and you know what, writing all of this out now, I realize we probably don't _need_ the `arc.tables` interface to expose the `arc-sessions` table, as all of that is handled via the `arc.http.sessions` interface. Though, I guess, technically this is different behaviour. Can probably skip adding that! Sorry for the previous request 🙈